scipy/signal/ltisys.py

```diff
@@ -445,6 +423,13 @@ def impulse(self, X0=None, T=None, N=None):
         """
         return impulse(self, X0=X0, T=T, N=N)
 
+    def impulse2(self, X0=None, T=None, N=None, **kwargs):
```

Comment:
-1 on adding a function if it's expected to be deprecated soon, it's setting people up for eventual disappointment :)